_includes/head.html

```diff
@@ -1,24 +1,6 @@
 <head>
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta http-equiv="Content-Security-Policy"  
-    content="default-src 'self';
-             connect-src 'self' https://*.tiles.mapbox.com https://api.mapbox.com https://osmstats-api.hotosm.org https://tasking-manager-tm4-production-api.hotosm.org https://matomo.hotosm.org https://hotosm.bamboohr.com;
-             font-src 'self' https://fonts.gstatic.com https://themes.googleusercontent.com;
-             frame-src https://embed.ted.com https://www.youtube.com https://docs.google.com;
-             frame-ancestors 'self';
-             img-src 'self' data: blob: https://s3.amazonaws.com http://hotwww.s3-website-us-east-1.amazonaws.com https://api.monosnap.com https://cdn.hotosm.org https://monosnap.com https://resources.bamboohr.com https://s3.amazonaws.com https://www.colorhexa.com;
-             object-src self;
-             manifest-src 'self';
-             media-src 'self';
-             worker-src blob:;
-             child-src blob: ;
-             script-src 'self' 'unsafe-inline' https://ajax.googleapis.com https://api.tiles.mapbox.com https://cdn.hotosm.org https://matomo.hotosm.org https://hotosm.bamboohr.com;
-             style-src 'self' 'unsafe-inline' https://api.tiles.mapbox.com https://fonts.googleapis.com https://hotosm.bamboohr.com">
-  <meta http-equiv="X-Content-Type-Options" content="nosniff">
-  <meta http-equiv="X-Frame-Options" content="SAMEORIGIN">
-  <meta http-equiv="X-XSS-Protection" content="1; mode=block">
-  <meta http-equiv="Strict-Transport-Security" content="max-age=63072000">
```

Comment:
Do we need to remove all these lines or just the problematic ones (ie CSP)?

I'm fine with either this will eventually get back in the codebase after more research.

Reply:
Meta tag security headers aren't standard. So some browsers ignore them and therefore, the observatory scores aren't swayed. https://observatory.mozilla.org/analyze/hotosm.org